share/spice/missing_kids/missing_kids.js

```diff
@@ -0,0 +1,60 @@
+(function (env) {
+    "use strict";
+    env.ddg_spice_missing_kids = function (api_result) {
+        
+        if (!api_result) Spice.failed('missing_kids');
+        if (api_result.error) Spice.failed('missing_kids');
```

Comment:
Can we combine those?  if api_result is a null object then attempting to access a state variable will throw an error, correct?  So line 5 ensures it is not null and 6 safely checks the error field.